test/benchmark.js

```diff
@@ -21,17 +21,37 @@ const isWin32 = require('os').platform() === 'win32';
 const npmScript = isWin32 ? 'npm.cmd' : 'npm';
 
 const testDir = resolve('.tmp-hexo-theme-unit-test');
+const zeroEksDir = process.env.TRAVIS_BUILD_DIR
+  ? join(process.env.TRAVIS_BUILD_DIR, '0x')
+  : resolve(testDir, '0x');
 const hexoBin = resolve(testDir, 'node_modules/.bin/hexo');
 
+const zeroEks = require('0x');
+
+let isProfiling = process.argv.join(' ').includes('--profiling');
+let isBenchmark = process.argv.join(' ').includes('--benchmark');
+
+if (!isProfiling && !isBenchmark) {
+  isProfiling = true;
+  isBenchmark = true;
+}
```

Comment:
Run both profiling and benchmark when neither `--profiling` nor `--benchmark` exists.